src/Integration.Tests/ArgsTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿using CSnakes.Runtime.Python;
+using System.ComponentModel;
+
+namespace Integration.Tests;
+public class ArgsTests : IntegrationTestBase
+{
+    [Fact]
+    public void PositionalOnly()
+    {
+        var mod = Env.TestArgs();
+        Assert.Equal(6, mod.PositionalOnlyArgs(1, 2, 3));
+    }
+
+    [Fact]
+    public void CollectStarArgs()
+    {
+        var mod = Env.TestArgs();
+        var td = TypeDescriptor.GetConverter(typeof(PyObject));
+
+        using (GIL.Acquire()) {
+            using PyObject? arg1 = td.ConvertFrom(3L) as PyObject;

```

Comment:
```suggestion
            using PyObject arg1 = (PyObject)td.ConvertFrom(3L);

```

Let's do an explict cast. Yes, it could result in a runtime cast exception, but that'd be better than a null reference exception on the Assert (and clean up the warnings in the compiler).